source/codegen/metadata/nifake_extension/enums.py

```diff
@@ -1,3 +1,20 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-FAKE API metadata version 1.2.0d9
-enums = {}
+enums = {
+    'LargeValue': {
+        'codegen_method': 'public',
+        'values': [
+            {
+                'name': 'ZERO',
+                'value': 0
+            },
+            {
+                'name': 'POSITIVE',
+                'value': 1073684512
+            },
+            {
+                'name': 'NEGATIVE',
+                'value': 3221168161
```

Comment:
Well we could change scrapigen to do that. I don't like the idea of the separate mappings, because it's really not needed since these are 32-bit values.

Reply:
I agree `generate-mappings` doesn't sound as good. If it's easy to fix in `scrapigen` I'd prefer that so that the metadata files are a valid-source-of-truth. If that doesn't work or isn't worth it, I'm alright with this or whatever y'all agree on. You could also consider heavy-handed-verbosity like an `overflow_to_i32` flag or something like that.

Reply:
Fixing in scrapigen.